app/src/app/handleLocations.js

```diff
@@ -38,7 +38,7 @@ export function openWindow(url) {
     webPreferences: {
       nodeIntegration: url.startsWith('file:'),
       nodeIntegrationInWorker: url.startsWith('file:'),
-      contextIsolation: !url.startsWith('file:'),
+      contextIsolation: false,
```

Comment:
@AleshaOleg shouldn't we want the isolation to be true all the time?

Reply:
@nikgraf that window if only to open local files which are built-in to the application, here it's not possible to set `contextIsolation` to true

Reply:
ahh ok